zerver/lib/webhooks/git.py

```diff
@@ -163,6 +190,22 @@ def get_commits_content(commits_data, is_truncated=False):
         ).replace('  ', ' ')
     return commits_content.rstrip()
 
+
 def get_short_sha(sha):
     # type: (Text) -> Text
     return sha[:7]
+
+
+def get_all_committers(commits_data):
+    # type: (List[Dict[str, Any]]) -> Dict[str, int]
+    committers = defaultdict(int)  # type: Dict[str, int]
+    for commit in commits_data:
+        committers[commit['name']] += 1
+    committers_names = list(sorted(committers.keys()))  # type: List[str]
```

Comment:
It looks your algorithm is not correct.
Consider the situation:
UserA made 3 commits
UserB made 4 commits
UserC made 5 commits
UserD made 6 commits
UserE made 7 commits

you sorted committers by name not by the number of commits so the result  will be: "UserA(3), UserB(4), others(X)" instead of "UserE(7), UserD(6), others(Y)"